testapi.pm

```diff
@@ -620,13 +624,19 @@ I<Make sure the command does not write to the serial output.>
 
 =cut
 sub assert_script_run {
-    my ($cmd, $timeout, $fail_message) = @_;
+    my ($cmd, $args, $fail_message) = @_;
+    unless (ref $args eq ref {}) {
```

Comment:
found it on https://rosettacode.org/wiki/Named_parameters#Perl
any other remarks?